skbio/stats/power.py

```diff
@@ -821,24 +830,31 @@ def _calculate_power(p_values, alpha=0.05):
 
     """
 
-    w = (p_values < float(alpha)).sum()/float(p_values.shape[0])
+    if p_values.ndim == 1:
+        p_values = p_values * np.array([[1]])
+
+    w = (p_values < float(alpha)).sum(axis=1)/float(p_values.shape[1])
```

Comment:
This isn't critical.

But this is completely equivalent to  `w = (p_values < alpha).sum(axis=1)/p_values.shape[1]`
since Python 3 division is being imported through `from __future__ import division`
So the `float` conversion is unnecessary